src/web3_utils.py

```diff
@@ -68,7 +69,9 @@ def get_strategy_from_vault(
     return strategy_contract, vault_contract
 
 
-def get_strategies_and_vaults(node: Web3, chain: str) -> Tuple[List[Dict], List[Dict]]:
+def get_strategies_and_vaults(
+    node: Web3, chain: str
+) -> Tuple[List[Contract], List[Contract]]:
     strategies = []
     vaults = []
```

Comment:
I would maybe add here explicit type so it's expected to have `vaults: List[Contract] = []`

Reply:
ah ok gotcha, good call